app/views/applicationstatus/how_to_pay.scala.html

```diff
@@ -0,0 +1,88 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import forms.Form2
+@import include._
+@import include.forms2._
+
+@(paymentReference: Option[String])(implicit request: Request[_],m:Messages, lang: Lang)
+
+@header = {
+    @components.back_link()
+    @headingWithPlaceholder(("howtopay.title", ""), "summary.status")
+}
+
+@main(
+    title = Messages("howtopay.title") + " - " + Messages("summary.howtopay"), heading = header ) {
+    <h2 class="heading-small">@Messages("howtopay.title2")</h2>
+    <p>@Messages("howtopay.para.1")</p>
+    <p>@Messages("howtopay.para.2")
+        <a id="your-messages" href="your-registration/your-messages">
+            @if(!paymentReference.getOrElse("").isEmpty) {
```

Comment:
will paymentReference.map(_.nonEmpty) work instead of !paymentReference.getOrElse("").isEmpty ?